1-exercises/E-for-loop/exercise1.js

```diff
@@ -8,6 +8,7 @@
 // Change the below code to use a for loop instead of a while loop.
 let i = 0;
 while(i < 26) {
+for (let i = 0; i < 26; i++) {
```

Comment:
Is your for loop nested inside the while loop? How come you have done it like this?